### ISSUE TITLE
missing the "fasta" DNA output file

### ISSUE THREAD

**Issue by andriangajigan** [faylward/viralrecall#26]
Thank you for creating this tool! Helps me a lot. 

I noticed I was missing this output file when I ran ViralRecall:
*.fasta: The DNA coding sequence of the predicted proteins from Prodigal

Running, for instance,
python viralrecall.py -i bin.10.fa -p bin.10 -t 8 -c

would only give me
bin.10.faa
bin.10.full_annot.tsv
bin.10.markerout
bin.10.pfamout
bin.10.summary.tsv
bin.10.vogout

**Comment by faylward** [comment]
Yes - "-a" gives the protein output and "-d" provides the nucleic acid
sequence. You have the right command.
Does that answer your question?

Frank

**Comment by andriangajigan** [comment]
Yes. Thanks Frank!
